### ISSUE TITLE
Ability to use EXIF DateTimeOriginal instead of current time

### ISSUE THREAD

**Issue by andreyvin** [marijnkampf/Random-Photo-Screensaver#56]
Almost every photoframe has ability to show date time image was taken.

Thanks for the source I was able to modify code to take date from exif data, remove time, put it into right corner and set font the same as time font in up-right corner.
It seems to be nice to have ability to put DateTimeOriginal from exif into Monitors->Clock with ability of format modification.

**Comment by marijnkampf** [comment]
You can set the metadata shown in the Metadata tab:
http://www.abscreensavers.com/random-photo-screensaver/configuration/?tab=metadata

If you want to relocate/style it see: Did you http://www.abscreensavers.com/random-photo-screensaver/faq/#custommonitorcss

**Comment by andreyvin** [comment]
Yes, I use it, but I want to show date in format YYYY-MM-DD
I remove hours and minutes from DateTimeOriginal in c# code but would like to have this out of the box using just configuration dialog.

**Comment by marijnkampf** [comment]
That makes sense. I'd have to find a generic way to enable formatting of metadata. I will add it to my to do list and will need to do some research and find out whether metadata dates are always in the same format etc.

PS. did you find it straight forward to compile RPS from source? If you ran into any issues please let me know and I can improve the instructions.

**Comment by andreyvin** [comment]
Yes it was easy. I've install the product, build solution in VS2013 and replace files.

Thanks for your project!
